sympy/simplify/_cse_diff.py

```diff
@@ -0,0 +1,255 @@
+"""Module for differentiation using CSE."""
+
+from sympy import cse, Matrix, Derivative, MatrixBase
+from sympy.utilities.iterables import iterable
+
+
+def _process_cse(replacements, reduced_expressions):
+    r"""
```

Comment:
```suggestion
    """
```

raw string only necessary if it contains some invalid characters.